editor/scaffolds/sidecontrol/sidecontrol-node-selection.tsx

```
@@ -1106,19 +1109,31 @@ function SelectedNodeProperties({
+                    />
+                  </PropertyLine>
+                ) : (
+                  <PropertyLine>
+                    <PropertyLineLabel>Radius</PropertyLineLabel>
+                    <CornerRadiusControl
+                      value={cornerRadius}
+                      onValueCommit={actions.cornerRadius}
+                    />
+                  </PropertyLine>
+                )}
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)